packages/remark-lint-blockquote-indentation/readme.md

```diff
@@ -15,19 +15,6 @@ too little.
 
 ## Contents
 
-*   [What is this?](#what-is-this)
```

Comment:
Hmmm, the table of contents plugin seems to be broken as well 🤔

Reply:
Should happen if you run `npm format` afterwards?

Reply:
Ah! That did the trick, thanks @wooorm!